cves/CVE-2014-3583.yml

```diff
@@ -118,11 +133,13 @@ discovered:
 
     If there is no evidence as to how this vulnerability was found, then you may
     leave this part blank.
-  answer:
-  date:
-  automated:
-  google:
-  contest:
+  answer: |
+    White hat hacker Teguh P. Alko discovered the vulnerability while searching for
+    security issues in HTTPD.
```

Comment:
Was he working for anyone in particular or was this just a personal project?

Reply:
Based off what I found he seems to just be an individual who does this kind of work as a personal project